src/systems/userinput/userinput.js

```diff
@@ -73,23 +73,37 @@ AFRAME.registerSystem("userinput", {
     this.registeredMappings = new Set([keyboardDebuggingBindings]);
     this.xformStates = new Map();
 
+    let connectedGamepadBindings;
```

Comment:
Ah yes.. We were not ending up with the `registeredMappings` we wanted when e.g. touch was connected but you had not entered VR. I'll incorporate these changes to some others that I am working on for responding to active device changes correctly.